### PR TITLE
Adjusted readme

### PR DESCRIPTION
**Pull request by jolo-dev**
> REPLACE THIS TEXT BLOCK
>
> Describe the reason for this change, what the solution is, and any
> important design decisions you made. 
>
> Remember to follow the [CONTRIBUTING GUIDE] and [DESIGN GUIDELINES] for any
> code you submit.
>
> [CONTRIBUTING GUIDE]: https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md
> [DESIGN GUIDELINES]: https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md

Closes #<issue number here>.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

---

<details open="true"><summary>Generated summary (powered by <a href="https://app.graphite.dev">Graphite</a>)</summary>

> # Pull Request Description
> 
> ## TL;DR
> This pull request refactors the `aws-codepipeline-actions` README.md file to use a declared `lambdaFunction` instead of creating a new one within the `CodeCommitSourceAction` declaration.
> 
> ## What changed
> The `CodeCommitSourceAction` declaration in the README.md file of `aws-codepipeline-actions` package was updated. Instead of creating a new `LambdaFunction` within the `target` field, it now uses a previously declared `lambdaFunction`.
> 
> ```diff
> -    target: new LambdaFunction(new Function(stack, 'TestFunction', {
> -      code: Code.fromInline('foo'),
> -      handler: 'bar',
> -      runtime: Runtime.NODEJS_LATEST,
> -    })),
> +    target: lambdaFuntion,
> ```
> 
> ## How to test
> To test this change, follow the instructions in the README.md file to set up a `CodeCommitSourceAction` using a `lambdaFunction`. Ensure that the pipeline works as expected with the `lambdaFunction` as the target.
> 
> ## Why make this change
> This change simplifies the code and makes it easier to understand. It also promotes the use of declared variables, which can be more efficient and easier to manage in larger codebases.
</details>

### CODE REVIEW

**Comment by jolo-dev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1** <a href="https://app.graphite.dev/github/pr/jolo-dev/aws-cdk/1?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #2** <a href="https://app.graphite.dev/github/pr/jolo-dev/aws-cdk/2?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/jolo-dev/aws-cdk/2?utm_source=stack-comment).

**Review by jolo-dev — COMMENTED** [review]
LGMT
